src/open_dread_rando/specific_patches/static_fixes.py

```diff
@@ -64,22 +61,25 @@ def _apply_one_sided_door_fix(scenario: Brfld, map_blockages: construct.Containe
     map_blockages[mirrored.sName] = mirrored_map
 
     # Add a reference to the other shield to the main actor
-    life_comp[direction] = f"Root:pScenario:rEntitiesLayer:dctSublayers:{layer_name}:dctActors:{mirrored.sName}"
+    life_comp[direction] = f"Root:pScenario:rEntitiesLayer:dctSublayers:{sublayer_name}:dctActors:{mirrored.sName}"
 
-    for group_name in scenario.all_actor_groups():
-        if any(scenario.is_actor_in_group(group_name, x, layer_name) for x in [actor_name, other.sName]):
-            for name in [actor_name, mirrored.sName, other.sName]:
-                scenario.add_actor_to_group(group_name, name, layer_name)
+    for x in [other.sName, mirrored.sName]:
+        editor.copy_actor_groups(
+            { "actor": actor_name },
+            { "actor": x },
+            scenario.name,
+        )
```

Comment:
A door will always be in the actor groups for both collision cameras it connects. In theory, the engine supports it only being in one, but in-game they have it in both so that it appears in both rooms. Because of that, I decided to simplify this code since we want both covers to be in both groups, which the door (`actor_name`) will always be in. I think this should also run faster.

Worth noting that when I tested in game, I didn't run into issues, but I also didn't play a full seed.

Reply:
I'm also pretty sure that adding a shield to only one side would make it behave as a "one-way", like how in vanilla the missile door between artaria screw attack and elevator can't be opened from the right.

Reply:
Good thing that's not what I'm doing here, haha